ext/web/03_abort_signal.js

```diff
@@ -120,6 +120,7 @@ class AbortSignal extends EventTarget {
     const signal = new AbortSignal(illegalConstructorKey);
     signal[timerId] = setTimeout(
```

Comment:
Note that Deno.core offers a `queueSystemTimer` to work around this sort of issue. The changes here will still be leaking a timer, as the timer is only ever cleared when the timer has fired (which should have already cleared the timer).